gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/rolesprovider/google/GoogleDirectoryUserRolesProvider.groovy

```diff
@@ -50,6 +54,43 @@ class GoogleDirectoryUserRolesProvider implements UserRolesProvider, Initializin
     Assert.state(config.credentialPath != null, "Supply an service account credentials path")
   }
 
+  private class GroupBatchCallback extends JsonBatchCallback<Groups> {
+
+    Map<String, Collection<String>> emailGroupsMap
+
+    String email
+
+    @Override
+    void onFailure(GoogleJsonError e, HttpHeaders responseHeaders) throws IOException {
+      throw new RuntimeException("Failed to fetch groups: " + e.getMessage())
```

Comment:
There's not really anything to recover from here, so rethrowing only makes something else deal with it. The best course of action is probably to log the exception and move on.